config/tendermint/config.go

```diff
@@ -79,6 +79,7 @@ func GetConfig(rootDir string) cfg.Config {
 	mapConfig.SetDefault("block_size", 10000)      // max number of txs
 	mapConfig.SetDefault("block_part_size", 65536) // part size 64K
 	mapConfig.SetDefault("disable_data_hash", false)
+	mapConfig.SetDefault("timeout_handshake", 10000)
```

Comment:
Does it make sense to specify units of time: either as a part of the name (`handshake_timeout_ms`) or comment `// ms` ?